vignettes/bayes_factors.Rmd

```diff
@@ -101,9 +101,6 @@ posterior <- as.data.frame(model)$group2
 
 insight::get_priors(model)
 ```
-```{r prior_n_post2, message=FALSE, warning=FALSE}
```

Comment:
@mattansb You don't agree that it's nicer to display kables in vignettes than the R printed output of dataframes?

Reply:
Oops, I'll add that back :)